lib/deck.rb

```diff
@@ -0,0 +1,37 @@
+class Deck
+  attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def rank_of_card_at(index)
+    @cards[index].rank
+  end
+
+  # Method high_ranking_cards should read rank of each card in the cards array, if the
+  # card rank is greater than 10, the card should go into the high_rank_cards array
+
+
+  def high_ranking_cards
+    @high_rank_cards = []
+    @cards.each do |card|
+      if card.rank > 10
```

Comment:
The spec indicates this should be 11 or above meaning we'd want this line to be `if card.rank >= 11`